changelog/2767.removal

```diff
@@ -0,0 +1,2 @@
+* remove the internal optional multi-typed attribute _evalskip
```

Comment:
typo: *replace*

Also please use correct grammar and formatting so we show users a well formatted changelog. 😉 

Also don't need to add a `*`, towncrier does it for us automatically.

```
Remove the internal multi-typed attribute ``Node._evalskip`` and replace it with the boolean ``Node._skipped_by_mark``.
```

Reply:
@RonnyPfannschmidt I was thinking this whole mark refactoring deserves a blog post on pytest's blog, to explain the pain points of the current design, our attempts to fix it and how it might affect users. People will be much more understanding if they know the reasons of why their test suite broke.

Reply:
The changelog is still the same, what do you think about my suggestion above @RonnyPfannschmidt?